sass/base/_css_variables.scss

```diff
@@ -2,7 +2,7 @@
     // Default Hex values to be replaced by applyTheme()
     // these are define to prevent flashes of unstyled content
     --away-indicator: #ffbc42;
-    --away-indicator-dark: #c79e3f;
+    s: #c79e3f;
```

Comment:
is this typo?

Reply:
Yeah, Got engaged in another task. Fixed now